packages/beacon-node/src/network/peers/score/score.ts

```diff
@@ -46,6 +46,16 @@ export class RealScore implements IPeerScore {
     this.setLodestarScore(newScore);
   }
 
+  applyReconnectionCoolDown(timeInMin: number): void {
+    // set banning period to time in ms in the future from now
+    this.lastUpdate = Date.now() + timeInMin * 60 * 1000;
+    // TODO: already applying banning period so existing scores were set to
+    // self-decay to zero after banning period. should we also enforce the decay
+    // from existing scores though @twoeths?
+    this.lodestarScore = -1;
```

Comment:
I don't think we should set lodestar score here, it's not fair to all peers, some peers with good behavior should take less time to decay to 0 than peers with bad behavior